### PR TITLE
OCS Hold session start/end counters

### PR DESCRIPTION
**Pull request by fholzhauser**

### CODE REVIEW

File: src/ergw_aaa_prometheus.erl [hunk 1]
```diff
@@ -0,0 +1,13 @@
+-module(ergw_aaa_prometheus).
+-export([declare/0]).
+
+declare() ->
+    prometheus_counter:declare([
+        {name, ocs_hold_session_start},
+        {labels, []},
+        {help, "Total number of OCS Hold sessions started"}]),
+    prometheus_counter:declare([
+        {name, ocs_hold_session_end},
```
Comment (RoadRunnr) [inline src/ergw_aaa_prometheus.erl]: this start/end thing feels strange. Why not have a single gauge for session in state. Make state a label and `ocs_free` one of the possible values.

Comment (hwinkel) [reply src/ergw_aaa_prometheus.erl]: I think the Reason for that is that Prometheus only knows numeric Values. Everything else are labels.?!?

Comment (fholzhauser) [reply src/ergw_aaa_prometheus.erl]: The idea was, that with these counters we can have both a snapshot of the current number of ocs free session at the moment (what you'd have with a gauge), and the count of such sessions in any period of time.

Comment (RoadRunnr) [reply src/ergw_aaa_prometheus.erl]: But the count over time should be handled by the metric (via a histogram), the prometheus collector or the prometheus server, not by the application.

Comment (RoadRunnr) [reply src/ergw_aaa_prometheus.erl]: also the name of metric does not match prometheus conventions

**Comment by fholzhauser** [comment]
I close this PR, as there is another, more generic, session stats solution is being worked on.

**Pull request closed without merging**